docs/UserGuide.md

```diff
@@ -203,9 +203,9 @@ the constraints of each parameter when used in a command.
 | `ADDRESS`          | Address of the patient.                                                                                    | - Any value is allowed. <br> - Should not be blank.                                                                                                                                                                                                                                                                                                                                    | :white_check_mark: `Orchard Road, Block 124, #02-01`                                                                              |
 | `PHONE`            | Phone number of the patient.                                                                               | - Should only contain numbers.<br> - Should be at least 3 digits long <br> - Should not be blank. <br> - There is no limit on the length of phone number accepted, in order to accommodate international phone numbers.                                                                                                                                                                | :white_check_mark: `98765432`<br>:x: `+65 9876 5432`                                                                              |
-| `ALLERGY`          | Allergy of the patient.                                                                                    | - Only alphanumeric characters are allowed.<br> - Should not exceed 30 characters in length <br> - Should not be blank.                                                                                                                                                                                                                                                                | :white_check_mark: `Peanuts`<br>:x: `Pe@nuts`                                                                                     |
+| `ALLERGY`          | Allergy of the patient.                                                                                    | - Only alphanumeric characters are allowed.<br> - Should not exceed 30 characters in length.<br> - Should not be blank.<br> - Multiple consecutive spaces between letters or words are not allowed.                                                                                                                                                                                    | :white_check_mark: `Peanuts`<br>:x: `Pe@nuts` <br>:x: `Pea  nuts`                                                                 |
```

Comment:
Perhaps we can rephrase to "Multiple consecutive spaces between letters and/or words are not allowed".
So that they may not be confused and think its just between letters only and between words only, because "addAllergy i/T1234567F c/t [SPACE][SPACE]   actualWord" may lead to an error as well. Just a nit!

Reply:
This applies for the error message as well, and the constraint for medCon

Reply:
Self resolved